tests/foreman/ui/test_contentview_new.py

```diff
@@ -0,0 +1,44 @@
+"""Test class for Content View UI
+
+:Requirement: Contentview
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: ContentViews
+
+:Assignee: ltran
```

Comment:
`:Assignee:` is not valid anymore, use `:team:` instead
```suggestion
:team: Phoenix-content
```